irc.html

```diff
@@ -492,14 +492,14 @@ <h5 class="title">
         </thead>
         <tbody>
         {{#data}}
-          <tr onclick="window.location='{{linkedin}}';">
+          <tr>
             <th scope="row">
                 <img src="https://res.cloudinary.com/dsxobn1ln/image/upload/c_scale,h_48,w_48/{{image}}" class="avatar">
             </th>
-                <td>{{name}}</td>
+                <td ><a target="_blank" href="{{linkedin}}">{{name}}</a></td>
```

Comment:
```suggestion
                <td><a target="_blank" href="{{linkedin}}">{{name}}</a></td>
```
Remove the extra space inside the 'td' tag

Reply:
ok